try:
    a=int(7)
    print("program burada")

except: #yanına alınan hangi hatada çalışacağı yazlılabilir. ör: except ValueError gibi.
    print("hata oluştu")

print("bloklar sona erdi.")